Puzzlepart/prosjektportalen365
# Added Phase parameter to new template

### Your checklist for this pull request


### Description

PR is part of the template rework, bugfixes and minor changes, 
Note: It is part of unreleased functionality, so it won't be a part of the changelog

### How to test

- [] 1. Set a value to Term set Ids different than the default value.
- [] 2. Create a project with the template
- [] 3. See that it works

### Relevant issues (if applicable)
Closes #630 

💔Thank you!
